ml-agents/mlagents/trainers/policy/policy.py

```diff
@@ -46,16 +43,17 @@ def __init__(
         self.vis_obs_size = sum(
             1 for shape in behavior_spec.observation_shapes if len(shape) == 3
         )
-        self.model_path = model_path
-        self.initialize_path = self.trainer_settings.init_path
-        self._keep_checkpoints = self.trainer_settings.keep_checkpoints
+        self.vis_obs_shape = (
```

Comment:
It does not seem that vis_obs_shape was present before. Is it being used ?

Reply:
This is added for the onnx export, where we need to create all the dummy input tensors with the right shape.
Tensorflow does not need this.